apps/condo/domains/acquiring/utils/validate/bankAccount.utils.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Validator BankAccount
+ *
+ * You need a BIC for validation
+ *
+ * The following checks are performed:
+ * 1) Сhecking for emptiness
+ * 2) Checking for length and format (Consists of 20 digits)
+ * 3) Сhecking checksum verification for BankAccount
+ */
```

Comment:
Its awesome that you added some docstring. But best is to add some examples. Also, am I right by saying that this validation only apply to RU ?